packages/main/src/plugin/color-registry.ts

```diff
@@ -808,4 +808,86 @@ export class ColorRegistry {
       light: colorPalette.black + '2',
     });
   }
+
+  // button
+  protected initButton(): void {
+    const button = 'button-';
+
+    this.registerColor(`${button}primary-bg`, {
+      dark: colorPalette.purple[600],
+      light: colorPalette.purple[500],
+    });
+    this.registerColor(`${button}primary-hover-bg`, {
+      dark: colorPalette.purple[500],
+      light: colorPalette.purple[600],
+    });
+    this.registerColor(`${button}secondary`, {
+      dark: colorPalette.gray[200],
+      light: colorPalette.purple[600],
+    });
+    this.registerColor(`${button}secondary-hover`, {
+      dark: colorPalette.purple[500],
+      light: colorPalette.purple[800],
+    });
+    this.registerColor(`${button}text`, {
+      dark: colorPalette.white,
+      light: colorPalette.white,
+    });
+    this.registerColor(`${button}disabled`, {
+      dark: colorPalette.charcoal[50],
+      light: colorPalette.gray[700],
+    });
+    this.registerColor(`${button}disabled-text`, {
+      dark: colorPalette.charcoal[50],
+      light: colorPalette.gray[900],
+    });
+    this.registerColor(`${button}danger-border`, {
+      dark: colorPalette.red[600],
+      light: colorPalette.red[600],
+    });
+    this.registerColor(`${button}danger-bg`, {
+      dark: colorPalette.charcoal[700],
+      light: colorPalette.gray[200],
+    });
+    this.registerColor(`${button}danger-text`, {
+      dark: colorPalette.white,
+      light: colorPalette.black,
+    });
+    this.registerColor(`${button}danger-hover-bg`, {
+      dark: colorPalette.charcoal[400],
+      light: colorPalette.gray[400],
+    });
```

Comment:
I would add another value four button-danger-hover-text and set it to white, or point to button-text for the hover state